### DISCUSSION TITLE
Exporting regular logs with RichHandler

### DISCUSSION THREAD

**Discussion by kfirdo** [Textualize/rich#1208] [Q&A]
When logging using RichHandler I want to export the logs from the console to a HTML file.
When calling export_html(), only what I print() or log() directly on the console instance is exported, the rest (logged from ```logging``` module), is displayed in the console but not in the exported file.
How can I also save the logged messages to the file?
Note: I do enjoy the simple formatting that logging gives me (The "INFO", "DEBUG" messages and the overflow of the text. That's why I don't use console.log()), if there is a simple solution using this approach I would be glad to hear about it.

**Comment by willmcgugan** [comment]
You can set the `console` argument so that `RichHandler` writes to the same console object.
